Makefile

```diff
@@ -27,10 +27,10 @@ test-cov:
 
 .PHONY: lint
 lint:
-	@PATH=$(shell go env GOPATH)/bin:$(PATH)
+	@PATH=$(shell go env GOBIN):$(PATH)
```

Comment:
this seemed to make some assumptions on workspace layout which were not true on my machine, feel free to ignore